README.md

```diff
@@ -26,6 +26,12 @@ If you prefer not to use a prepackaged binary, or want to have the latest unrele
 * Run `go get github.com/taskcluster/livelog`
 * Run `go get github.com/taskcluster/taskcluster-proxy`
 
+Note that the build process requires custom tooling, and your `go get` may mention errors.
```

Comment:
Nice spot! Indeed that was broken, but luckily we can fix it, so I would prefer to replace your line 29 with the following change to line 25:

* Run `go get -tags nativeEngine github.com/taskcluster/generic-worker` (all platforms) and/or `go get -tags dockerEngine github.com/taskcluster/generic-worker` (linux only)

The reason for this is, for the first build, this is entirely sufficient, since no code changes have been made, so assuming CI has passed on the master branch, the additional checks that build.sh makes should have already been made (e.g. code has been generated, tests compile, etc etc).

Reply:
Fixed!